sli/slistartup.cc

```diff
@@ -133,9 +140,25 @@ SLIStartup::GetenvFunction::execute( SLIInterpreter* i ) const
   i->EStack.pop();
 }
 
+static std::string getsliprefix() {
+#if defined(__linux__) || defined(__APPLE)
+    Dl_info info;
+    if (dladdr((void*)getsliprefix, &info)) {
+        char const * soname = info.dli_fname;
+        std::string result = std::string(soname);
```

Comment:
```suggestion
       std::string result( info.dli_fname );
```
Can you find a more informative name than `result`?